spotify/Makefile

```diff
@@ -1,3 +1,7 @@
 .PHONY: run dev
 run dev:
-	python3 main.py
\ No newline at end of file
+	python3 main.py
+
+.PHONY: docker-build
+docker-build:
+	docker build -t openpodcast/spotify-cron .
```

Comment:
why is the name cron? currently there is no cron, do we add it later?

Reply:
Yes, I'll add that part. However I'm still not sure about the image name.

Reply:
maybe just spotify-connector?

Reply:
Check. Changed it to `spotify-connector`.